### PR TITLE
SSJ - Verification of Basic Vacancy Information section while creating an NCI Vacancy

### PR DESCRIPTION
**Pull request by iamdez99**

### CODE REVIEW

File: src/test/java/ServiceNow/PlatformBusinessApps/SSJ/playwright/Features/OWM_Vacancy_Manager.feature [hunk 1]
```diff
@@ -1,36 +1,49 @@
...
-    And User confirms being a POC for this vacancy
+    And User "Holly Gemar-Griffith" confirms being a POC for this vacancy after selecting Yes for being the point of contact for this vacancy
     And User sets an "Open Date" entry as today's date
-    And User sets a "Close Date" entry
+    And User sets a "Close Date" entry a month from the Open Date
     And User sets "Scoring Due By Date" entry
-    And User verifies "Cover Letter" application document is marked as optional by default
-    And "Add More" button is displayed for Application Documents section
-    And User can add another application document
+#    #REVIEW ABOVE STEP WITH BA
```
Comment (lfcabrera14) [inline src/test/java/ServiceNow/PlatformBusinessApps/SSJ/playwright/Features/OWM_Vacancy_Manager.feature:21]: Where the above steps reviewed with BA? If so maybe the comment can be deleted - cabreralf

File: src/test/java/ServiceNow/PlatformBusinessApps/SSJ/playwright/StepsImplementation/OWM_Vacancy_Manager_StepsImpl.java
Comment (lfcabrera14) [inline src/test/java/ServiceNow/PlatformBusinessApps/SSJ/playwright/StepsImplementation/OWM_Vacancy_Manager_StepsImpl.java:1]: method on line 215 doesn't have a Javadoc comment - cabreralf
image

**Review by lfcabrera14 — CHANGES_REQUESTED** [review]
please review comments thanks - cabreralf

File: src/test/java/ServiceNow/PlatformBusinessApps/SSJ/playwright/StepsImplementation/OWM_Vacancy_Manager_StepsImpl.java
Comment (iamdez99) [reply src/test/java/ServiceNow/PlatformBusinessApps/SSJ/playwright/StepsImplementation/OWM_Vacancy_Manager_StepsImpl.java:1]: Added and updated PR.

Comment (lfcabrera14) [reply src/test/java/ServiceNow/PlatformBusinessApps/SSJ/playwright/StepsImplementation/OWM_Vacancy_Manager_StepsImpl.java:1]: thanks!

**Comment by lfcabrera14** [comment]
comments addressed and reviewed - cabreralf

**Pull request closed without merging**